src/main/java/seedu/recruittrackpro/logic/parser/FindCommandParser.java

```diff
@@ -26,8 +26,11 @@ public FindCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME);
 
+        String preamble = argMultimap.getPreamble();
+        boolean shouldContainAll = preamble.equals("--contain-all");
+
         if (!hasAtLeastOnePrefixPresent(argMultimap, PREFIX_NAME)
-                || !argMultimap.getPreamble().isEmpty()) {
+                || (!preamble.isEmpty() && !shouldContainAll)) {
```

Comment:
Just some nits for readability
```suggestion
boolean hasValidPrefix = hasAtLeastOnePrefixPresent(argMultimap, PREFIX_NAME);
boolean hasInvalidPreamble = !preamble.isEmpty() && !shouldContainAll;

if (!hasValidPrefix || hasInvalidPreamble) {